fixes.inc

```diff
@@ -5758,6 +5799,26 @@ native BAD_GetWeaponName(weaponid, weapon[], len) = GetWeaponName;
 		//  END:   OnPlayerEnterVehicle
 		// =============================
 
+		// =============================
+		//  BEGIN: OnPlayerEnterVehicle_3
+		// =============================
+		#if FIX_OnPlayerEnterVehicle_3
+			if (FIXES_gsPlayerBools[playerid] & e_FIXES_BOOLS_VEH_ENTER) {
```

Comment:
https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/b3ab260be1cf5989508d6c6a4ca6e3ed9fc8a33c and https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/30cb65981ed54208a3474a50c8646b5e4158816d